graphblas_algorithms/cluster.py

```diff
@@ -0,0 +1,60 @@
+from collections import OrderedDict
+
+import graphblas as gb
+from graphblas import Matrix, Vector, binary, select
+from graphblas.semiring import any_pair, plus_pair
+from networkx.utils import not_implemented_for
+
+
+def single_triangle_core(G, index):
+    M = Matrix(bool, G.nrows, G.ncols)
+    M[index, index] = False
+    C = any_pair(G.T @ M.T).new(name="C")  # select.coleq(G, index)
```

Comment:
Check it out, @jim22k, a use case for `coleq` select op!

Hmm, the comment is wrong.  Should be `select.coleq(G.T, index)` or `select.col(G.T == index)`.